setuptools/command/build_py.py

```diff
@@ -192,6 +192,28 @@ def get_package_dir(self, package):
         if self.distribution.src_root is not None:
             return os.path.join(self.distribution.src_root, res)
         return res
+    
+    def find_package_modules(self, package, package_dir):
+        modules = orig.build_py.find_package_modules(self, package, package_dir)
+        if not self.distribution.include_package_data:
```

Comment:
How is it that the `include_package_data` is relevant to whether modules are included in package?